### ISSUE TITLE
Feedback collector CG777

### ISSUE THREAD

**Issue by ThomasssArt** [kodadot/nft-gallery#10732]
First feedback from a collector:

![image](https://github.com/user-attachments/assets/a402779d-868c-4252-a881-5f1571331eff)

Tbh i also experience exatly the things mention above by myself...

Related issue:
- https://github.com/kodadot/ops-internal/issues/1816

**Comment by exezbcz** [comment]
yeah the ui is not really snappy, especially on mobile. Was the collector on mobile or desktop?

**Comment by ThomasssArt** [comment]
> yeah the ui is not really snappy, especially on mobile. Was the collector on mobile or desktop?

Desktop
